perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -573,6 +573,39 @@ public Void call(ApiCaller ac, Deserializer params) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Assign new sponsor to the selected sponsored members of another sponsor
+	 *
+	 * @param sponsored List<MemberWithSponsors>
+	 * @param copyFrom int id of the existing sponsor user
+	 * @param copyTo int id of the new user to be sponsor
+	 * @param copyValidity boolean whether to copy validity
+	 * @param validityTo String the last day, when the sponsorship is active, yyyy-mm-dd format.
+	 *
+	 */
+	copySponsoredMembers {
+		@Override
+		public Void call (ApiCaller ac, Deserializer params) throws PerunException {
+			params.stateChangingCheck();
+
+
+			boolean copyValidity = false;
+			if (params.contains("copyValidity")) {
+				copyValidity = params.readBoolean("copyValidity");
+			}
+			LocalDate validityTo = null;
+			if (params.contains("validityTo")) {
+				validityTo = params.readLocalDate("validityTo");
+			}
+
```

Comment:
a nitpick - i think we could check that `copyValidity` and `validityTo` are not both set at once and have separate javadocs in this file for them